packages/govern-server/src/core/errors.ts

```diff
@@ -16,8 +12,12 @@ export class ErrorException extends Error {
 
     // We define these as non-enumarable to prevent them
     // from appearing with the error in the console.
-    defineNonEnumerable(this, 'name', name)
-    defineNonEnumerable(this, 'code', code)
+    this.defineNonEnumerable(this, 'name', name)
+    this.defineNonEnumerable(this, 'code', code)
+  }
+
+  private defineNonEnumerable(instance: object, name: string, value: any) {
+    Object.defineProperty(instance, name, { value, enumerable: false })
   }
```

Comment:
I think we could move this function elsewhere, as it’s pretty generic? But if we want to keep it here, we could use `this` directly rather than passing it as a parameter.